src/Components/Common/AvatarEditModal.tsx

```diff
@@ -114,11 +114,18 @@ const AvatarEditModal = ({
 
     setIsProcessing(true);
     setIsCaptureImgBeingUploaded(true);
-
-    await handleUpload(selectedFile, () => {
+    try {
+      await handleUpload(selectedFile, () => {
+        setSelectedFile(undefined);
+        setPreview(undefined);
+        setPreviewImage(null);
+      });
+    } catch (e) {
+      console.error(e);
+    } finally {
       setIsCaptureImgBeingUploaded(false);
       setIsProcessing(false);
-    });
+    }
```

Comment:
These changes are not required, just calling onError() in the handler should be enough

Reply:
Done, I removed them.
Is it ok now?